utils/fft-eval/Makefile

```diff
@@ -0,0 +1,40 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fft-eval
+PKG_VERSION:=2017-06-28
+PKG_RELEASE=$(PKG_SOURCE_VERSION)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://github.com/simonwunderlich/FFT_eval.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=3cc175570379da172b0b2bcdbb8d2a42f83dad88
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz
+PKG_MIRROR_HASH:=fdda73b54e2ff115155d6c461d7600269045dfb9c5bb0d57a10a4e9ae7f669c2
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/$(PKG_NAME)
+	SECTION:=utils
+	CATEGORY:=Utilities
+	MAINTAINER:=Nicolas Pace <nicopace@altermundi.net>
+	URL:=https://github.com/simonwunderlich/FFT_eval
+	TITLE:=Evaluates FFT samples from ath9k driver
+	DEPENDS:= +libc +@PACKAGE_ATH_DEBUG +@PACKAGE_ATH_SPECTRAL
```

Comment:
Don't do ```+@```, this is considered bad practice. Use ```@``` instead.

Reply:
Hi @diizzyy why `+@` is considered bad practice?